src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,71 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import pages.BasePage;
+
+import static org.openqa.selenium.By.cssSelector;
+
+public class HomePage extends BasePage {
+    String newPlayListName = "New Playlist";
+
+    public HomePage(WebDriver givenDriver){
+        super(givenDriver);
+    }
+    //locators
+    @FindBy(css = "button[title='Delete this playlist']")
+    WebElement xButton;
+    @FindBy(css = "input[name='name']")
+    WebElement textField;
+    @FindBy(css = "img.avatar")
+    WebElement avatarIcon;
+    @FindBy(css = "li>a.active")
+    WebElement newPlayList;
+    @FindBy(css = "#nprogress")
+    WebElement successMessage;
+    @FindBy(css = "div.success.show")
+    static WebElement notificationMessage;
+    @FindBy(xpath = "//*[@id=\"playlists\"]/ul/li[3]/a")
+    WebElement playList;
+    @FindBy(xpath = "//*[@id=\"playlists\"]/ul/li[3]/nav/ul/li[1]")
+    WebElement editOption;
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```
   @FindBy(css = "img.avatar")
    private WebElement avatar
```